index.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 const decamelize = require('decamelize');
 
-const humanizeString = input => {
+const humanizeString = (input, opts = {}) => {
 	if (typeof input !== 'string') {
 		throw new TypeError('Expected a string');
 	}
 
-	input = decamelize(input);
-	input = input.toLowerCase().replace(/[_-]+/g, ' ').replace(/\s{2,}/g, ' ').trim();
+	input = opts.originalCase ? input : decamelize(input);
```

Comment:
I think the option name should be `preserveCase`.

Reply:
yes, this is much better, thanks